templates/taxi/manufacturer_list.html

```diff
@@ -7,7 +8,10 @@ <h1>
       Create
     </a>
   </h1>
-
+  <form method="get" action="" class="form-inline">
+    {{ search_form|crispy }}
```

Comment:
The 'crispy' filter is applied to the search form, which will render it with the crispy forms styling.